components/ProtegeProgramCard.tsx

```diff
@@ -0,0 +1,51 @@
+import {Box, Stack, Typography} from '@mui/joy';
+import Card from '@mui/joy/Card';
+import DarkOutlineButton from './DarkOutlineButton';
+import Image from 'next/image';
+
+export default function ProtegeProgramCard({ icon, type, description }: any) {
+  return (
+    <Card
+      variant='outlined'
+      className='border-round'
+      sx={{
+        width: {xs: '300px', lg: '350px'},
+        height: 'auto',
+        border: '2px solid #93AED3',
+        backgroundColor: 'white',
+        transition: 'all 0.2s ease',
+        '&:hover': {
+          boxShadow: '0px 4px 8px rgba(38, 38, 38, 0.2)',
+          top: '-4px',
+          border: '2px solid #93AED3',
+        }
```

Comment:
For the hover effect, you can use className='shadow-hover' instead to keep things consistent. It's defined in globals.css

Reply:
i changed it and it does however have a bit of difference from the previous hover effect but this looks good too